dev-python/qtconsole/qtconsole-4.1.1.ebuild

```diff
@@ -29,6 +29,10 @@ DEPEND="${RDEPEND}
 	)
 	test? (
 		>=dev-python/nose-0.10.1[${PYTHON_USEDEP}]
+		|| (
+			dev-python/PyQt4[${PYTHON_USEDEP},svg,testlib]
+			dev-python/PyQt5[${PYTHON_USEDEP},svg,testlib]
```

Comment:
fine with me, but please swap the entries still (most preferred should be the first in a `|| ( )` list)